cmd/backup/main.go

```diff
@@ -612,6 +644,39 @@ func (s *script) pruneOldBackups() error {
 		}
 	}
 
+	// Prune WebDAV backups
+	if s.webdavClient != nil {
+		candidates, err := s.webdavClient.ReadDir(s.c.WebdavPath)
+		if err != nil {
+			return fmt.Errorf("pruneOldBackups: error looking up candidates from remote storage: %w", err)
+		}
+		var matches []fs.FileInfo
+		var lenCandidates int
+		for _, candidate := range candidates {
+			s.logger.Infof("Candidate: %s", candidate.Name())
```

Comment:
Is this a stray log used for debugging?

Reply:
I thougt you'd log that too for minio....
I will delete it -- There is no added value from this log.
Deleted in: e18ec1f8989378d28be11e4096f1cd5413a862bb